src/js/gi/components/search/EligibilityForm.jsx

```diff
@@ -72,19 +71,27 @@ export class EligibilityForm extends React.Component {
             visible={this.props.eligibility.dropdowns.giBillChapter}
             onChange={this.props.eligibilityChange}/>
 
-        <If condition={
-            this.props.eligibility.militaryStatus === 'active duty' &&
-            this.props.eligibility.giBillChapter === '33'}>
-          <div className="militaryStatusInfo form-group">
+        {
+          this.props.eligibility.militaryStatus === 'active duty' &&
+          this.props.eligibility.giBillChapter === '33' &&
+          (<div className="militaryStatusInfo form-group">
             <i className="fa fa-warning"></i>
             <a title="Post 9/11 GI Bill"
                 href="http://www.benefits.va.gov/gibill/post911_gibill.asp"
                 id="anch_378" target="_blank">
             Post 9/11 GI Bill</a> recipients serving on Active Duty (or
             transferee spouses of a servicemember on active duty) are not
             eligible to receive a monthly housing allowance.
-          </div>
-        </If>
+          </div>)
+        }
+
+        {
+          this.props.eligibility.giBillChapter === '31' && (<div>
+            To apply for VR&E benefits, please <a
+                href="http://vabenefits.vba.va.gov/vonapp/main.asp"
```

Comment:
Is this the only place where this form is posted online? As we're trying to shut off VONAPP, not sure if we want to direct more traffic to it.